test/Microsoft.AspNet.Mvc.ViewFeatures.Test/ViewComponents/ViewComponentMethodSelectorTest.cs

```diff
@@ -1,282 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
The code formerly referred to as `ViewComponentMethodSelector` is part of the descriptor provider now.